server/main.lua

```diff
@@ -64,18 +64,16 @@ local function GetActiveData(cid)
 end
 
 local function IsPoliceOrEms(job)
-	for k, v in pairs(Config.PoliceJobs) do
-           if job == k then
-              return true
-            end
-         end
-         
-         for k, v in pairs(Config.AmbulanceJobs) do
-           if job == k then
-              return true
-            end
-         end
-    return false
+    local s,r = pcall(function()
+        local _job = type(job) == "string" and job or tostring(job)

```

Comment:
Please make this a guard clause instead. In this case, if a table is provided as the argument, it can basically try to convert it to a string. So check if it is a string and print something out from the guard clause if it is not. This is to make this prone to errors in the future.

Reply:
i dont see why you will pass a table to a job, this pr is from july, i dont undesrtand why is active at this point

Reply:
Me neither, but what I am saying is that it is possible since Lua is not a typed language.